### DISCUSSION TITLE
Sguil and Squert

### DISCUSSION THREAD

**Discussion by nehahussain** [Security-Onion-Solutions/securityonion#8110] [Unsupported Versions] [answered]
Can anyone guide me how I could install both of these,  need to use these to do my labs.

**Comment by xfaith** [comment]
Recommend finding a copy of the older unsupported copy of SecurityOnion 16.04 which has both of these applications as part of its baseline.  
I do not believe that they are compatible with 2.3

**Comment by nehahussain** [comment]
Do you have 16.04 version by any chance?
I am unable to find , every link takes me back to the current version.

**Comment by dougburks** [comment, marked as answer]
@nehahussain Security Onion 16.04 reached End Of Life over a year ago:
https://docs.securityonion.net/en/2.3/eol.html

We don't provide downloads or any support for it at this point. 

If your lab instructor really wants you to use old outdated software, then they should probably provide the software for you.
